src/main/java/uk/gov/companieshouse/filinghistory/consumer/delta/MappingUtils.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.filinghistory.consumer.delta;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public final class MappingUtils {
+
+    private MappingUtils() {}
+
+    static String getValueFromField(final JsonNode node, final String field) {
```

Comment:
It wasn't immediately clear from outside of this class what the method was doing. Could it be more descriptively named please? Something like ```getFieldValueFromJsonNode```?